MonstersAttacks.cs

```diff
@@ -1,31 +1,32 @@
-﻿using System;
+﻿using Castle_Fayyt.RNG;
+using System;
 
 namespace Castle_Fayyt
 {
-  class M6Attacks
+  internal class M6Attacks
   {
-    static RandomAttackPool<int> M6Claws = new RandomAttackPool<int>();
-    static RandomAttackPool<int> M6Bite = new RandomAttackPool<int>();
+    private static readonly RandomBag<int> m6Claws = new RandomBag<int>();
+    private static readonly RandomBag<int> m6Bite = new RandomBag<int>();
 
     public static int M6Damage()
     {
       bool M6Instantiated = false;
       if (M6Instantiated == false)
       {
-        M6Claws.AddEntry(0, 10);
-        M6Claws.AddEntry(1, 10);
-        M6Claws.AddEntry(8, 19);
-        M6Claws.AddEntry(12, 30);
-        M6Claws.AddEntry(16, 19);
-        M6Claws.AddEntry(20, 10);
-        M6Claws.AddEntry(50, 2);
-        M6Bite.AddEntry(0, 8);
-        M6Bite.AddEntry(1, 7);
-        M6Bite.AddEntry(5, 10);
-        M6Bite.AddEntry(7, 30);
-        M6Bite.AddEntry(9, 22);
-        M6Bite.AddEntry(11, 18);
-        M6Bite.AddEntry(20, 5);
+        m6Claws.AddEntry(0, 10);
```

Comment:
No, I left it this way because it looks like you're going to repeat the same switch-block pattern that you did in Arcana.cs and Attack.cs. I don't want to step on your toes.

However, I do think that the pattern you're using ought to be re-thought.